x-pack/plugins/canvas/shareable_runtime/types.ts

```diff
@@ -52,7 +50,7 @@ export interface CanvasRenderable {
   state: 'ready' | 'error';
   value: {
     as: string;
-    containerStyle: ContainerStyle;
+    containerStyle: any;
```

Comment:
Is this `ContainerStyle` type not importable from somewhere?

Reply:
the comment says `// @ts-expect-error Unlinked Webpack Type`. No idea what it means, but `tsc` uses `any` instead